mw-webapp/src/logic/wayPage/WayPage.tsx

```diff
@@ -59,7 +59,10 @@ export const WayPage = (props: WayPageProps) => {
             onChangeFinish={(text) => changeWayName(way, text)}
             isEditable={true}
           />
-          <DayReportsTable wayUuid={props.uuid} />
+          <DayReportsTable
+            wayUuid={props.uuid}
+            way={way}
```

Comment:
why does Days Reports Table should know anything about the way? It is enough information inside the records. Get rif of useless props and data duplication